backend/src/api/app.ts

```diff
@@ -188,6 +202,21 @@ app.use(
   matomoProxy
 );
 
+app.use(
+  '/pe',
+  async (req, res, next) => {
+    // Only allow specific users to access
+    const user = (await auth.authorize({
+      authorizationToken: req.cookies['crossfeed-token']
```

Comment:
I would change this to pull from the header as we can just use that instead of needing the cookie

https://github.com/cisagov/crossfeed/blob/dcb2d02ea00b202de61ae7afbf59f42cb0876064/backend/src/api/app.ts#L67-L69